### PR TITLE
Add macros and stylesheet attributes to web-component

### PR DESCRIPTION
**Pull request by tani**
Hi Michael, I added `macros` and `stylesheet` attributes to the component.
We can use the component with similar API of CLI by this request.
For example,

```html
<latex-js macros="https://example.com/macros.js" stylesheet="https://example.com/style.css">
\documentclass{article}
\begin{document}
Hello!
\end{document}
</latex-js>
```

Cheers!

### CODE REVIEW

File: src/cli.ls [hunk 1]
```diff
@@ -52,7 +52,7 @@ program
     # options about LaTeX and style
     .option '-c, --class <class>',      'set a default documentclass for documents without a preamble', 'article'
     .option '-m, --macros <file>',      'load a JavaScript file with additional custom macros'
-    .option '-s, --style <url>',        'specify an additional style sheet to use (can be repeated)', addStyle
+    .option '-s, --stylesheet <url>',   'specify an additional style sheet to use (can be repeated)', addStyle
```
Comment (michael-brade) [inline src/cli.ls]: good idea, but then you also need to change line 78

**Review by michael-brade — CHANGES_REQUESTED** [review]

**Comment by michael-brade** [comment]
Another good idea, thanks! You know, if you want, I could add you as a contributor, then you could push to your own branches, and if only little changes need to be made, I could fix it and merge it right away. What do you think?

File: src/cli.ls (hunk 1 above)
Comment (fgborges) [reply src/cli.ls]: Okay, wait a minute. I'll update it soon.

**Comment by fgborges** [comment]
Yes, I would like to join this exiting project. Of course, I can work on the branch in this repository.

File: src/cli.ls [hunk 2]
```diff
@@ -76,7 +76,7 @@ if program.macros
 
 
 if program.body and (program.style or program.url)
```
Comment (michael-brade) [inline src/cli.ls]:
```suggestion
if program.body and (program.stylesheet or program.url)
```

File: src/cli.ls [hunk 3]
```diff
@@ -76,7 +76,7 @@ if program.macros
...
-    console.error "error: conflicting options: 'url' and 'style' cannot be used with 'body'!"
+    console.error "error: conflicting options: 'url' and 'stylesheet' cannot be used with 'body'!"
```
Comment (michael-brade) [inline src/cli.ls]: LOL :grin: this is good, but it still lacks line 78 :)

**Review by michael-brade — CHANGES_REQUESTED** [review]

File: src/cli.ls (hunk 3 above)
Comment (fgborges) [reply src/cli.ls]: Thanks :laughing:

**Review by michael-brade — APPROVED** [review]

**Pull request merged**